auto_update_github_action.yml

```
@@ -56,3 +56,11 @@ jobs:
         CLOUDFLARE_ACCOUNT_EMAIL: ${{ secrets.CLOUDFLARE_ACCOUNT_EMAIL }}
         CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
         CLOUDFLARE_LIST_ITEM_LIMIT: ${{ secrets.CLOUDFLARE_LIST_ITEM_LIMIT }}
+
+    - name: Send ping request
+      if: env.PING_URL != ''
+      working-directory: cloudflare-gateway-pihole-scripts
+      env:
+        PING_URL: ${{ secrets.PING_URL }}
+      run: |
+        node -e "var https=require('https');https.get('${{ env.PING_URL }}').on('error',err=>console.log('Ping failed: '+err));"
```

Why not keep it simple and just use curl?

```suggestion
        curl "${{ env.PING_URL }}"
```